### ISSUE TITLE
Azure CLI stopped working after update: ImportError: DLL load failed while importing win32file: The specified procedure could not be found.

### ISSUE THREAD

**Issue by SeanFeldman** [Azure/azure-cli#25144]
> ### `az feedback` auto-generates most of the information requested below, as of CLI version 2.0.62

Cannot generate feedback with `az feedback`, all `az` commands are failing.

**Related command**
`az`

**Describe the bug**
Getting the following error:

```
Traceback (most recent call last):
  File "runpy.py", line 196, in _run_module_as_main
  File "runpy.py", line 86, in _run_code
  File "D:\a\_work\1\s\build_scripts\windows\artifacts\cli\Lib\site-packages\azure/cli/__main__.py", line 14, in <module>
  File "D:\a\_work\1\s\build_scripts\windows\artifacts\cli\Lib\site-packages\azure/cli/core/telemetry.py", line 19, in <module>
  File "D:\a\_work\1\s\build_scripts\windows\artifacts\cli\Lib\site-packages\azure/cli/telemetry/__init__.py", line 9, in <module>
  File "D:\a\_work\1\s\build_scripts\windows\artifacts\cli\Lib\site-packages\portalocker/__init__.py", line 4, in <module>
  File "D:\a\_work\1\s\build_scripts\windows\artifacts\cli\Lib\site-packages\portalocker/portalocker.py", line 11, in <module>
ImportError: DLL load failed while importing win32file: The specified procedure could not be found.
```

**To Reproduce**
Type `az`

**Expected behavior**
Should work

**Environment summary**

Install Method MSI (azure-cli-2.44.1.msi)
Windows 10 OS version
Powershell

**Comment by yonzhan** [comment]
@jiasli for awareness

**Comment by SeanFeldman** [comment]
I tried uninstalling and re-installing, didn't help.
The culprit seems to be the remaining files under the installation folder, `%ProgramFiles(x86)%\Microsoft SDKs\Azure\CLI2\Lib`, where not all files are removed. I had to manually delete `%ProgramFiles(x86)%\Microsoft SDKs\Azure\CLI2` after uninstalling and reinstalling Azure CLI fixed the issue.

Are all packaged upgraded correctly when updating Azure CLI?
